easybuild/easyconfigs/t/torchtext/torchtext-0.16.2-foss-2023b-PyTorch-2.1.2.eb

```
@@ -0,0 +1,51 @@
+easyblock = 'PythonPackage'
+
+name = 'torchtext'
+version = '0.16.2'
+local_pytorch_version = '2.1.2'
+versionsuffix = '-PyTorch-%s' % local_pytorch_version
+
+homepage = 'https://github.com/pytorch/text'
+description = "Data loaders and abstractions for text and NLP"
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+
+# Sources are no longer available in PyPI, it only has wheels
+source_urls = ['https://github.com/pytorch/text/archive/']
+sources = [{'download_filename': 'v%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}]
+patches = [
+    {'name': 'torchtext-0.14.1_use-system-libs.patch', 'alt_location': 'PyTorch-bundle'},
+    {'name': 'torchtext-0.16.2_download-to-project-root.patch', 'alt_location': 'PyTorch-bundle'},
+]
+checksums = [
+    {'torchtext-0.16.2.tar.gz': '6574b012804f65220329a2ad34a95c18e4df0d4cff2f862fb7862d57b374b013'},
+    {'torchtext-0.14.1_use-system-libs.patch': '1366d10c4755b6003194f7313ca11d165a80a13d325bee9d669ea2b333d82536'},
+    {'torchtext-0.16.2_download-to-project-root.patch':
+     '9d5599a9983729cf1fc7ab2a2f65d1887f223f528e15662ba1b4a5e359c9686d'},
+]
+
+builddependencies = [
+    ('RE2', '2024-03-01'),
+    ('binutils', '2.40'),
+    ('CMake', '3.27.6'),
+]
+
+dependencies = [
+    ('Python', '3.11.5'),
+    ('SciPy-bundle', '2023.11'),
+    ('SentencePiece', '0.2.0'),
+    ('tqdm', '4.66.2'),
+    ('PyTorch', local_pytorch_version),
+    ('double-conversion', '3.3.0'),
+    ('utf8proc', '2.9.0'),
+    ('torchdata', '0.7.1', '-PyTorch-%s' % local_pytorch_version),
+]
+
+download_dep_fail = True,
+use_pip = True
+sanity_pip_check = True,
```

not sure what the trailing commas are doing here, works by accident, so let's get rid of them
```suggestion
download_dep_fail = True
use_pip = True
sanity_pip_check = True